src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
```

Comment:
```suggestion
  const COST_PER_DAY = 40;
```
follow constants naming convention: 
```
const HOURS_IN_DAY = 24;
const hoursRemaining = currentHour - HOURS_IN_DAY;
```

Reply:
What differrent with upper case use with camelcase?